shell/platform/android/android_surface_gl_impeller.cc

```diff
@@ -75,10 +75,9 @@ static std::shared_ptr<impeller::Context> CreateImpellerContext(
     FML_LOG(ERROR) << "Could not add reactor worker.";
     return nullptr;
   }
-  FML_LOG(ERROR) << "Using the Impeller rendering backend.";
+  FML_LOG(ERROR) << "Using the Impeller GL rendering backend.";
```

Comment:
I guess the same comment about this being an info log.

Reply:
(At some point this shoul dbecome an info log but let's handle that in a separate patch)